ee/hogai/eval/ci/eval_memory_onboarding.py

```diff
@@ -0,0 +1,397 @@
+- No citation anywhere (i.e. link in parentheses)
+
+A FAIL occurs if ANY of the following is true:
+- Missing or incorrect opening format (doesn't start with "__Here's what I found on...:__")
+- No #### section headings found
+- Poor organization with wall-of-text paragraphs instead of structured sections
+- Contains follow-up suggestions like "Would you like to know more?" or "Contact us"
+- Sections lack bullet points or clear structure
+- Generic formatting without the specified markdown structure
+- A citation link is found
+- There is any follow-up suggestion for the reader (like "Let me know if you'd like X" or "To do something, go to Y")
+
+Note: The format requirements are specifically defined in the memory initialization prompt and must be followed exactly.
+
+Input: {{input}}
+Output: {{output}}
+
+Does the output follow the exact required formatting structure with proper opening, #### headers, and bullet points? Be brutal
+- pass: Follows all required formatting specifications exactly
+- fail: Missing required formatting elements or includes prohibited content""",
+            choice_scores={
+                "pass": 1.0,
+                "fail": 0.0,
+            },
+            model="gpt-4.1",
+            **kwargs,
+        )
+
+
+@pytest.fixture(scope="module")
+def core_memory():
+    """We override the conftest.py core_memory fixture, which has autouse=True, as for memory onboarding a CoreMemory must be ABSENT."""
+    pass
+
+
+@pytest.fixture
+def call_memory_onboarding():
+    """Fixture to call MemoryOnboardingNode with a parametrized EventTaxonomyItem as input."""
+
+    async def callable(input: tuple[str, Optional[EventTaxonomyItem]]) -> Optional[AssistantMessage]:
+        organization_name, taxonomy_item = input  # The org name MAY matter as it gets included in the system prompt
+        # Because memory is stateful at the team level, each eval case here must have its own team instance
+        _, team, user = await database_sync_to_async(User.objects.bootstrap)(
+            organization_name=organization_name, email=f"{uuid4()}@example.com", password=None
+        )
+        conversation = await Conversation.objects.acreate(team=team, user=user)
+
+        graph = (
+            AssistantGraph(team=team, user=user)
+            .add_memory_onboarding(AssistantNodeName.END)
+            .compile(checkpointer=False)
+        )
+        raw_state = await graph.ainvoke(
+            AssistantState(messages=[HumanMessage(content=SLASH_COMMAND_INIT)]),
+            # Mock the _retrieve_context method to return our input data via `configurable`
+            # Because evals run concurrently and everything's async, patch() just doesn't work
+            {"configurable": {"thread_id": conversation.id, "_mock_memory_onboarding_context": taxonomy_item}},
+        )
+
+        state = AssistantState.model_validate(raw_state)
+        if not state.messages:
+            return None
+        last_message = state.messages[-1]
```

Comment:
Question: Do we expect the last message to be the scrapped content or the verification prompt (`SCRAPING_VERIFICATION_MESSAGE`)?